### PR TITLE
Adding source Tenmanga

### PR DESCRIPTION
**Pull request by E3FxGaming**
Requested in #3275 and brought up again in  #3535.
I'm not a particular fan of the Tenmanga website, but it's better than adding the broken copycat Mangadeep.

### CODE REVIEW

**Comment by SnakeDoc83** [comment]
There's also http://www.manga99.com/, and I think all 3 sites are getting content from taadd.com. Taadd's design doesn't seem too far off from them either (at a cursory glance). Also, I think Taadd's content is already available by using the NineManga and NineAnime extensions. 

Taadd should be looked at, there's no open request for it but it's been mentioned before. Also need to see if there are any other sites using this theme (and if so, are they all clones which so far they seem to be). Basically, would like to decide whether this should be a factory extension before merging.

**Comment by MerradOklom** [comment]
https://www.niadd.com/list/Hot-Book/
https://www.taadd.com/list/Hot-Book/
https://www.nineanime.com/list/Hot-Book/
These three sites seems to be sharing the same theme. As mentioned, NineAnime got an extension not long ago, so maybe it would be better (and probably less time consuming?) to tweak it a bit to make it a Factory instead of creating a new one. Also TenManga is using the same theme as NineManga, so maybe you could add it to that one and make it a multi-source extension?

**Comment by E3FxGaming** [comment]
> would like to decide whether this should be a factory extension before merging

I would like to work on this factory, if you decide to do it like that.
I would just need to know what it should be called and on which existing extension or factory extension it should be based on.

Summarizing:

The factory should try to cover the following websites:
- http://www.tenmanga.com
- http://mangadeep.com/
- http://www.manga99.com
- https://www.taadd.com
- https://www.niadd.com
- https://www.nineanime.com
- http://en.ninemanga.com
- http://es.ninemanga.com
- http://br.ninemanga.com
- http://ru.ninemanga.com
- http://de.ninemanga.com
- http://it.ninemanga.com

File: src/en/tenmanga/build.gradle [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+ext {
+    extName = 'Tachiyomi: Tenmanga'
```
Comment (arkon) [inline src/en/tenmanga/build.gradle:5]: Don't need to have the `Tachiyomi: ` part anymore.

**Comment by E3FxGaming** [comment]
I removed the "Tachiyomi:" from the extension name. Could someone take another look at this?

**Comment by SnakeDoc83** [comment]
@E3FxGaming Sorry, tried to reply before but Github wouldn't post the message and I didn't circle back.

I think a factory extension might be the way to go and I would call it MangaYes. From what I can tell they may have been the progenitor of the theme (it originally looked like WieManga, which uses a file named mangayes.css). MangaYes currently forwards to Taadd. 

I would start with the sources below. Although NineManga may fit with them, you'd have to completely rewrite it to make the new sources fit in. I think it makes more sense to start with a cleaner base and not worry about NineManga for now. That said, feel free to look at NineManga, NineAnime, and WieManga for some potentially useful information.

http://www.tenmanga.com
http://mangadeep.com/
http://www.manga99.com
https://www.taadd.com
https://www.niadd.com

**Comment by E3FxGaming** [comment]
Ok, then should I close this pull request and create a new pull request once I've got something to show?

**Comment by AbdullahM0hamed** [comment]
You could do that or make this PR a draft until you have refactored it.

**Pull request closed without merging**